frontend/packages/operator-lifecycle-manager/src/components/operand/operand-link.tsx

```diff
@@ -6,7 +6,8 @@ import { ClusterServiceVersionModel } from '../../models';
 
 export const OperandLink: React.SFC<OperandLinkProps> = (props) => {
   const { namespace, name } = props.obj.metadata;
-  const csvName = () =>
+  const csvName =
+    props.csvName ||
```

Comment:
not part of this PR but would be good to use `csvNameFromWindow` here for `|| ` condition